src/components/general/Page.module.css

```diff
@@ -39,6 +39,8 @@
 
 .frosted {
   backdrop-filter: blur(10px);
+  position: relative;
+  z-index: 10;
```

Comment:
why did we need this?

Reply:
should have been called out in the commit message for this one. It was this crazy thing where we had a 5 pixel bar that stole focus as you hovered down autocomplete results